index.html

```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
         </ul>
       </div>
     </section>
+    <section class="works-container" id="works">
+      <div class="works-header-container header-area">
+        <div class="works-header">
+          <h2>My recent works</h2>
+          <div class="line"></div>
+        </div>
+        <div class="works-header-multipost">
+          <img class="img-placeholder" src="./img/placeholders/Img Placeholder.png" alt="placeholder-image">
+          <div class="works-stories">
+            <h2>Multi-Post Stories</h2>
```

Comment:
- [OPTIONA] Kindly consider aligning this heading to the start as given in the design guidelines. See the images below👇

Your design | Given design
---| ---|
![image](https://github.com/Xecyc/My-Portfolio-Project/assets/106119105/170df7e6-3f13-4ed5-917e-2a39c1d73e0b) | ![image](https://github.com/Xecyc/My-Portfolio-Project/assets/106119105/ff8ff9c3-3bb3-4330-b903-5e3237b35ea6)